src/languages/en-US.ts

```diff
@@ -333,6 +333,7 @@ export default class extends Language {
 				users: 'Users',
 				guilds: 'Guilds',
 				channels: 'Channels',
+				steve: 'Steve Version',
```

Comment:
Immediately after I approved this PR I remembered that there's a botName variable that comes from the config file, implement that here and then we should be good for a merge

Reply:
Resolved by [3e43031](https://github.com/stevebot-project/steve/pull/67/commits/3e43031919d021869895cd4b61bf914cfdf98189)